Python/AlgorithmComparisons.py

```diff
@@ -15,19 +15,19 @@
 random_numbers_reverse_sorted = sorted(random_numbers, reverse=True)
 
 
-def print_comparison(comparison):
-    largest_name_len = len(max(comparison.keys(), key=len))
+def print_comparison(comparison) -> None:
+    largest_name_len: int = len(max(comparison.keys(), key=len))
```

Comment:
I think we can get rid of type annotations where the variable name is more than enough to make the reader understand the type. Let's remove the type annotations in places where it is not required.

Reply:
This is not intended for the reader but instead the static type checker. This makes sure the variable type is not changed whenever it gets reassigned. 

Try using PyRight on it

Reply:
Okay cool then.